backend/app/__init__.py

```diff
@@ -4,11 +4,11 @@
 import sentry_sdk
 
 if "REDIS_TLS_URL" in os.environ:
-    db = redis.Redis.from_url(os.environ["REDIS_TLS_URL"], ssl_cert_reqs=None)
+    db = redis.Redis.from_url(os.environ["REDIS_TLS_URL"], ssl_cert_reqs=None, decode_responses=True)
```

Comment:
This seems good for cases where we're storing strings, but what about when we're storing JSONs? I guess `json.loads` both works with bytes and strings right?

Reply:
Actually, as far as I know `json.loads` only works with strings. I'm surprised we haven't had to decode bytes responses from Redis till now.